docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,182 @@
+
+The location of the configuration files, can vary depending on the particular installation.
+
+After configuring the Kafka cluster, few topics must be created, which are required by the `Airy apps`. Those topics can be found in the bash script `infrastructure/scripts/provision/create-topics.sh`. The script can be run, after setting properly the environment variables for the particular Kafka installation: 
+- `ZOOKEEPER` (default: airy-cp-zookeeper:2181)
+- `PARTITIONS` (default: 10)
+- `REPLICAS` (default: 1)
+
+Deploying Kafka on docker is not recommended for production environments with multiple brokers and under heavy load. However, we have a provided a way to deploy the whole kafka cluster on top of kubernetes with helm.
+
+To deploy kafka on kubernetes with helm, after setting the proper kubernetes config file and context, you must run:
+```sh
+helm install airy infrastructure/helm-chart/charts/kafka/
+```
+
+By default, the `Schema registry` deployment is created with `replicas=0` because we want to ensure Zookeeper and Kafka are running properly beforehand, to avoid errors and crashing of this app.
+
+You can customize the deployment, by editing the configuration files in `infrastructure/helm-chart/charts/kafka/charts`.
+
+### PostgreSQL
+
+A cluster of PostgreSQL servers can be used or a managed one such as AWS RDS.
+If deployed on kubernetes, consider running the app on a StatefulSet workload, for persistent storage.
+
+After the server has been provisioned, a database must be created with proper credentials (username and password), required later when starting the `Airy apps`.
+
+We have provided a helm chart, which can deploy PostgreSQL server in kubernetes, after setting the proper kubernetes config file and context. For creating the PostgreSQL server with helm, set the `pgPassword` variable in `infrastructure/helm-chart/charts/postgres/values.yaml` file and run:
+```sh
+helm install postgres infrastructure/helm-chart/charts/postgres/
+```
+
+### Redis
+
+A cluster of redis servers can be used or a managed one such as AWS ElastiCache for Redis
+If deployed on kubernetes, consider running the app on a StatefulSet workload, for persistent storage.
+
+We have provided a helm chart, which can deploy the Redis server in kubernetes, after setting the proper kubernetes config file and context. For creating the redis cluster with helm run:
+```sh
+helm install redis infrastructure/helm-chart/charts/redis/
+```
+
+### Kubernetes or docker runtime
+
+The `Airy apps` run on top of kubernetes, which is our preferred runtime. All of the apps can also be deployed using docker, however we recommend kubernetes for more robust and reliable deployment in production environments.
```

Comment:
as noted in other parts of the document, we have to be more explicit about this. If it's a requirement (and probably at the moment it should be so it simplifies getting started with the platform), we should say clearly

Reply:
I suggest we `recommend` and state that this guide is about deploying on `Kubernetes`. However, it is fully possible to deploy this also on docker.